.github/workflows/publish.yml

```diff
@@ -6,20 +6,57 @@ jobs:
   main:
     name: Publish
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
 
     steps:
-      - uses: actions/checkout@v1
+      - name: Checkout repo
+        uses: actions/checkout@v3
 
       - name: Use Node.js 18.x
         uses: actions/setup-node@v2.2.0
         with:
-          node-version: '18.x'
+          node-version: "18.x"
 
       - name: Install
         run: yarn
 
-      - name: format
+      - name: Format
         run: yarn format
 
       - name: Check formatting
-        run: git diff --exit-code
\ No newline at end of file
+        run: git diff --exit-code
+
+      # TODO: Remove and modify "Create Catalog" once argo-tasks released
+      - name: Get Collections
+        run: |
+          echo "$(find stac/ -name collection.json)" > files.txt
+
+      # catalog.json is not pushed to the repository (temporary solution)
+      - name: Create Catalog
+        uses: docker://ghcr.io/linz/argo-tasks:v2
```

Comment:
I thought t he latest version of this just took the path to the stac folder? or has it not been merged yet?

Reply:
It has been merged. I'll update that

Reply:
`stac-catalog --output stac/catalog.json --template template/catalog.json stac/` errors with 
```
Error: FilePaths are not relative base: stac/ file: /github/workspace/stac/auckland/auckland_2010-2012_0.5m/rgb/2193/collection.json
```
I passed the full absolute path to workaround it
`stac-catalog --output stac/catalog.json --template template/catalog.json /github/workspace/stac/`